src/gui/render.cpp

```diff
@@ -30,171 +30,188 @@
 
 #include <sys/types.h>
 
-#include "video.h"
-#include "render.h"
-#include "setup.h"
 #include "control.h"
-#include "mapper.h"
 #include "cross.h"
 #include "hardware.h"
-#include "support.h"
+#include "mapper.h"
+#include "render.h"
+#include "setup.h"
 #include "shell.h"
 #include "string_utils.h"
+#include "support.h"
 #include "vga.h"
+#include "video.h"
 
 #include "render_scalers.h"
 
 Render_t render;
 ScalerLineHandler_t RENDER_DrawLine;
 
-static void RENDER_CallBack( GFX_CallBackFunctions_t function );
+static void RENDER_CallBack(GFX_CallBackFunctions_t function);
 
-static void Check_Palette(void) {
+static void Check_Palette(void)
+{
 	/* Clean up any previous changed palette data */
 	if (render.pal.changed) {
 		memset(render.pal.modified, 0, sizeof(render.pal.modified));
 		render.pal.changed = false;
 	}
-	if (render.pal.first>render.pal.last) 
+	if (render.pal.first > render.pal.last)
 		return;
 	Bitu i;
 	switch (render.scale.outMode) {
-	case scalerMode8:
-		break;
+	case scalerMode8: break;
 	case scalerMode15:
 	case scalerMode16:
-		for (i=render.pal.first;i<=render.pal.last;i++) {
-			uint8_t r=render.pal.rgb[i].red;
-			uint8_t g=render.pal.rgb[i].green;
-			uint8_t b=render.pal.rgb[i].blue;
-			uint16_t newPal = GFX_GetRGB(r,g,b);
+		for (i = render.pal.first; i <= render.pal.last; i++) {
+			uint8_t r       = render.pal.rgb[i].red;
+			uint8_t g       = render.pal.rgb[i].green;
+			uint8_t b       = render.pal.rgb[i].blue;
+			uint16_t newPal = GFX_GetRGB(r, g, b);
```

Comment:
may a new line before `newPal`, so the r/g/b alignment tightens up (given all three are related, but not of the same type as newPal?)

Reply:
Yeah I agree.